docs/consumer/Receiver.md

```diff
@@ -1,6 +1,81 @@
 # Stake Receiver
 
 The Stake Receiver is on the consumer side and connected an External Staker on the Provider side.
-This handles the normalization of the token streams and communicates with the Meta-Staker.
+This handles the normalization of the token streams and communicates with "Virtual Staking".
+It is connected to the Provider chain via IBC and "receives" the various staking commands
+
+**TODO** This was called the "Consumer contract" in previous versions and probably needs a better name
+
+## Setup
+
+When we [deploy the contracts](../ibc/Overview.md#deployment), we connect the Stake Receiver on the consumer chain
+with an [External Staking](../provider/ExternalStaking.md) contract on the Provider. Once this connection is established,
+Consumer governance can authorize this Stake Receiver with some ability to mint on the ["Virtual Staking" contract](./VirtualStaking.md).
```

Comment:
Perhaps worth noting how the connection is established?

Reply:
This is updated, but the deployment process is defined in this link: [deploy the contracts](../ibc/Overview.md#deployment)